examples/native/components/ControlExamples/Array/Array.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react-native';
+import { Array } from './Array';
+
+const ArrayMeta: ComponentMeta<typeof Array> = {
+  title: 'Array',
```

Comment:
The other controls seem to have "{name} control" titles, so it could be a bit tidier if this was uniform with them ("Array control").

Reply:
Makes sense, I'll update this.